TUTORIAL.md

```diff
@@ -1,51 +1,45 @@
-
-This tutorial shows you how to build a microservices architecture with [JHipster 4.3.0](https://jhipster.github.io/2017/04/13/jhipster-release-4.3.0.html). 
-You'll generate a gateway (powered by Netflix Zuul and the JHipster Gateway), a microservice (that talks to MongoDB), and use Docker Compose to make
-sure it all runs locally. Then you'll deploy it to Minikube and Google Cloud using Kubernetes.
-
-{:toc}
-
-## What is JHipster?
+// Why is this set up like a quote, and why are you opening with it?
 
 JHipster is one of those open-source projects you stumble upon and immediately think, "`Of course!`" It combines three 
 very successful frameworks in web development: Bootstrap, Angular, and Spring Boot. Bootstrap was one of the first dominant 
 web-component frameworks. Its largest appeal was that it only required a bit of HTML and it worked! All the efforts we 
 made in the Java community to develop web components were shown a better path by Bootstrap. It leveled the playing field 
 in HTML/CSS development, much like Apple's Human Interface Guidelines did for iOS apps.
 
-JHipster was started by Julien Dubois in October 2013 (Julien's first commit was on 
-[October 21, 2013](https://github.com/jhipster/generator-jhipster/commit/c8630ab7af7b6a99db880b3b0e2403806b7d2436)). The 
-first public release (version 0.3.1) was launched December 7, 2013. Since then, the project has had over 130 releases! 
-It is an open-source, Apache 2.0-licensed project on GitHub. It has a core team of 19 developers and over 320 contributors. 
-You can find its homepage at [http://jhipster.github.io](http://jhipster.github.io/). If you look at 
-[the project on GitHub](https://github.com/jhipster/generator-jhipster), you can see it's mostly written in JavaScript 
-(39%) and Java (27%).
-
 At its core, JHipster is a [Yeoman](http://yeoman.io/) generator. Yeoman is a code generator that you run with a `yo` 
 command to generate complete applications or useful pieces of an application. Yeoman generators promote what the Yeoman 
 team calls the "`Yeoman workflow`". This is an opinionated client-side stack of tools that can help developers quickly 
 build beautiful web applications. It takes care of providing everything needed to get working without the normal pains 
 associated with a manual setup.
 
+*Learn more about JHipster, including it's origin, at [http://jhipster.github.io](http://jhipster.github.io).*
+
+This tutorial shows you how to build a microservices architecture with [JHipster 4.3.0](https://jhipster.github.io/2017/04/13/jhipster-release-4.3.0.html). You'll generate a gateway (powered by Netflix Zuul and the JHipster Gateway), a microservice (that talks to MongoDB), and use Docker Compose to make sure it all runs locally. Then you'll deploy it to Minikube and Google Cloud using Kubernetes.
+
+// Reworked the intro, lmk what you think!
+
+{:toc}
+// what does toc indicate here?
```

Comment:
This macro works when using Asciidoctor. I was testing to see if it works with Markdown. It doesn't. ;)